### ISSUE TITLE
services.guix does not exist

### ISSUE THREAD

**Issue by SohamG** [NixOS/nixpkgs#308327]
https://github.com/NixOS/nixpkgs/blob/576ecd43d3b864966b4423a853412d6177775e8b/nixos/modules/services/misc/guix/default.nix#L53

I am trying to add `services.guix.enable = true;` to my nixos config, but it doesn't find the option (23.11). I believe the line above should be `cfg = with lib; ...`  as later on the module checks `lib.mkIf cfg.enable` but defines the `enable` option on `options.services.guix` instead of `config.services.guix`. 

Will report back if if I get around to testing this fix.

**Comment by SohamG** [comment]
CC @foo-dogsquared

**Comment by SohamG** [comment]
Woops my bad its been a while since I wrote a module...

Regardless, I realized that `services/misc/guix` did not exist in my nixpkgs outPath. I did some digging on the github and "solved" this issue by setting my system flake as follows:
```
    nixpkgs.url = "github:nixos/nixpkgs/nixos-23.11";
```
instead of `github:nixos/nixpkgs/23.11` (ie the 23.11 tag). Don't know if this is user error on my part or something bigger. 

Marking as closed.
